src/pages/pricing.tsx

```diff
@@ -1,263 +1,312 @@
+                <div className="tw-mb-sm md:tw-mb-0 tw-col-span-full md:tw-col-start-2 md:tw-col-span-5">
+                    <PricingPlan
+                        name="Business"
+                        price="$99 per active user/month"
+                        description="Full platform access for teams and orgs, all on a dedicated Cloud instance."
+                        features={BIZ_FEATURES_OVERVIEW}
+                        isEnterprise={false}
+                        buttons={<BusinessCTA btnOnMobile={true} />}
+                    />
                 </div>
 
-                <ContentSection className="tw-max-w-screen-sm">
-                    <Blockquote
-                        border={false}
-                        quote="Sourcegraph gives us the ability to search for and refactor
-                        references to deprecated services, libraries, URL patterns, and more across
-                        our 2000+ repositories, and the confidence that we're not leaving anyone
-                        behind."
-                        author="Aneesh Agrawal, Software Engineer, Lyft"
-                        logo={{
-                            src: '/external-logos/lyft-logo.svg',
-                            alt: 'Lyft logo: Read the case study',
-                            href: '/case-studies/criteo-tackles-big-code',
-                        }}
+                <div className="tw-col-span-full md:tw-col-start-7 md:tw-col-span-5">
+                    <PricingPlan
+                        name="Enterprise"
+                        price="Custom pricing"
+                        description="Enterprise-grade security, scale, and support with custom deployment options."
+                        features={ENTERPRISE_FEATURES_OVERVIEW}
+                        isEnterprise={true}
+                        buttons={<EnterpriseCTA btnOnMobile={true} />}
                     />
-                </ContentSection>
+                </div>
+            </ContentSection>
 
-                <div className="row">
-                    <div className="mb-4 tw-mx-auto col-md-12">
-                        <div className="px-6 card">
-                            <h2 className="my-3 tw-font-semibold">Team</h2>
+            <h2 className="tw-text-center tw-mx-auto tw-mb-5 tw-max-w-2xl">
+                The code intelligence platform for modern dev teams
+            </h2>
+            <CustomerLogos />
 
-                            <div className="row">
-                                <div className="mb-4 tw-pr-md tw-mx-auto tw-text-xl col-md-6">
-                                    If you have more than 10 users, you can upgrade to the Team plan. This includes all
-                                    of the features of the Free plan, plus:
+            <ContentSection parentClassName="tw-px-0 md:tw-px-sm">
+                <table className="tw-relative tw-border-0">
```

Comment:
Do you think we could get the table to shrink a bit further on mobile? It seems to break the viewport around 525px. I'm thinking we could allow the first two columns in the table and then allow overflow-x scrolling so users can scroll the table horizontally.

![Screen Shot 2022-09-21 at 2 26 51 PM](https://user-images.githubusercontent.com/1733936/191584033-4733b12b-262f-458b-93e5-41a4e87feae2.png)

Reply:
overflow-x disables the table header from being sticky on scroll. What about removing `Compare all features` title at that size? I think that'd look more clean, there's not that much on the table otherwise, just a lot of rows

Reply:
Ok, let's try that and see how it looks 👀

Reply:
Ok I removed the title & added overflow-x auto b/c on really small screens it was still acting up. Take another look. Header is sticky on md & up screens, sm & down scrolls horizontal to keep layout happy

Reply:
Nice! That looks good and the table doesn't break the viewport width. It's too bad those sticky headers can't stay on mobile though while you scroll. Think you could whip up a solution for that? Don't want to block this page though if it's going to take a lot more time.

Reply:
Just took a look! That looks better and the table doesn't break the viewport width. It's too bad we lose those sticky headers on mobile while you scroll though. As a user, I think that would be nice to have. Personally, I like to conveniently compare things when I view tables on mobile sites, so I think others comparing our features would like that too. Think you could whip up a solution for that? Don't want to block this page though if it's going to take a lot more time.